pandas/__init__.py

```diff
@@ -56,6 +56,7 @@
 from pandas.util.print_versions import show_versions
 from pandas.io.api import *
 from pandas.util._tester import test
+from pandas.formats.style import Styler
```

Comment:
I do think it needs to be importable somewhere 'public', not sure if top level is the right spot, could argue either way.  Do we need a `pandas.api.io`?